docs/03-tutorials/00-api-exposure/apix-09-expose-and-secure-workload-with-certificate.md

```diff
@@ -0,0 +1,352 @@
+
+     ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE/$MTLS_GATEWAY_NAME
+   export CLIENT_CERT_CRT_FILE=client.example.com.crt
+	 export CLIENT_CERT_CSR_FILE=client.example.com.csr
+	 export CLIENT_CERT_KEY_FILE=client.example.com.key 
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com.
+
+2. Expose an instance of the HttpBin service on mTLS Gateway by creating an APIRule CR in your Namespace. Run:
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   ---
+   apiVersion: gateway.kyma-project.io/v1beta1
+   kind: APIRule
+   metadata:
+     name: httpbin-mtls-gw-unsecured
+     namespace: ${NAMESPACE}
+   spec:
+     host: httpbin-vs.${DOMAIN_TO_EXPOSE_WORKLOADS}
+     service:
+       name: httpbin
+       port: 8000
+     gateway: ${GATEWAY}
+     rules:
+       - path: /.*
+         methods: ["GET"]
+         accessStrategies:
+           - handler: noop
+         mutators:
+           - handler: noop
+       - path: /post
+         methods: ["POST"]
+         accessStrategies:
+           - handler: noop
+         mutators:
+           - handler: noop
+   EOF
+   ```
+
+3. Generate Client certificate signed by Client Root CA: 
+
+   ```bash
+   # Create a new key and CSR for the client certificate
+   openssl req -out ${CLIENT_CERT_CSR_FILE} -newkey rsa:2048 -nodes -keyout ${CLIENT_CERT_KEY_FILE} -subj "/CN=client.example.com/O=example"
+   # Sign the client cert with CA cert
+   openssl x509 -req -days 365 -CA ${CLIENT_ROOT_CA_CRT_FILE} -CAkey ${CLIENT_ROOT_CA_KEY_FILE} -set_serial 0 -in ${CLIENT_CERT_CSR_FILE} -out ${CLIENT_CERT_CRT_FILE}
+   ```
+  
+  </details>
+  <details>
+  <summary>
+  Function
+  </summary>
+  
+1. Export the following values as environment variables:
+
+     ```bash
+   export DOMAIN_TO_EXPOSE_WORKLOADS={DOMAIN_NAME}
+   export GATEWAY=$NAMESPACE/$MTLS_GATEWAY_NAME
+   export CLIENT_CERT_CRT_FILE=client.example.com.crt
+	 export CLIENT_CERT_CSR_FILE=client.example.com.csr
+	 export CLIENT_CERT_KEY_FILE=client.example.com.key 
+   ```
+   >**NOTE:** `DOMAIN_NAME` is the domain that you own, for example, api.mydomain.com
+
+2. Expose a sample Function on mTLS Gateway by creating an APIRule CR in your Namespace. Run:
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   ---
+   apiVersion: gateway.kyma-project.io/v1beta1
+   kind: APIRule
+   metadata:
+     name: function-mtls-gw-unsecured
+     namespace: ${NAMESPACE}
+   spec:
+     host: function-example.${DOMAIN_TO_EXPOSE_WORKLOADS}
+     service:
+       name: function
+       port: 80
+     gateway: ${GATEWAY}
+     rules:
+       - path: /.*
+         methods: ["GET"]
+         accessStrategies:
+           - handler: noop
+         mutators:
+           - handler: noop
+       - path: /post
+         methods: ["POST"]
+         accessStrategies:
+           - handler: noop
+         mutators:
+           - handler: noop
+   EOF
+   ```
```

Comment:
Same as previous:
API rule is not needed. It's confusing. Only ISTIO CRDs should be used.